Algorithms/Introduction/intro_algorithms_answers.txt

```diff
@@ -0,0 +1,30 @@
+Calculate Tip(bill)
+  tip = 0
+  total_bill = bill
+  if bill > 5 && bill < 10
+    total_bill = bill + tip
+  else
+    total_bill = (bill * 0.20)
+    # round down to nearest dollar for convenience
+  end
+  return total_bill
```

Comment:
While this is totally fine, I prefer my pseudocode to be less... code. Instead of operators, I like to state things like `SET total_bill TO bill`, for example. Your comment about rounding is actually pretty exemplary pseudocode.